packages/dds/tree/src/test/utils.ts

```diff
@@ -614,6 +622,29 @@ export function validateSnapshotConsistency(
 	expectSchemaEqual(treeA.schema, treeB.schema, idDifferentiator);
 }
 
+/**
+ * {@link JsonableTree}s can contain handles, which do not compare well with assert.deepEqual.
```

Comment:
Nit: the description here implies the use case, but doesn't spell it out explicitly. It might be worth updating to explicitly call out the use case, then explain the _why_ of it.

Reply:
Done.